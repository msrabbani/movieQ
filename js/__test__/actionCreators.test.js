// @flow

import {setSearchTerm, addAPIData} from '../actionCreators';

test('setSearchTerm', () => {
  expect(setSearchTerm('New York')).toMatchSnapshot();
});

test('addAPIData', () => {
  expect(
    addAPIData({
      title: 'Breaking Bad',
      year: '2008â€“2013',
      description: "A high school chemistry teacher diagnosed with inoperable lung cancer turns to manufacturing and selling methamphetamine in order to secure his family's future.",
      poster: 'bb.jpg',
      imdbID: 'tt0903747',
        trailer: 'XZ8daibM3AE',
        rating: '7.9'
    }),
  ).toMatchSnapshot();
});
