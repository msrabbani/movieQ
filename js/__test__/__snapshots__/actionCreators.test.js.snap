// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addAPIData 1`] = `
Object {
  "payload": Object {
    "description": "A high school chemistry teacher diagnosed with inoperable lung cancer turns to manufacturing and selling methamphetamine in order to secure his family's future.",
    "imdbID": "tt0903747",
    "poster": "bb.jpg",
    "rating": "7.9",
    "title": "Breaking Bad",
    "trailer": "XZ8daibM3AE",
    "year": "2008â€“2013",
  },
  "type": "ADD_API_DATA",
}
`;

exports[`setSearchTerm 1`] = `
Object {
  "payload": "New York",
  "type": "SET_SEARCH_TERM",
}
`;
